tests/scene/test_code_edit.h

```diff
@@ -3972,6 +3972,24 @@ TEST_CASE("[SceneTree][CodeEdit] Backspace delete") {
 	code_edit->backspace();
 	CHECK(code_edit->get_text() == "line 1\nline 2\nline 3");
 
+	SUBCASE("[CodeEdit] folding") {
+		code_edit->set_line_folding_enabled(true);
+
+		/* Unfold previous folded line on backspace if caret is at the first column. */
```

Comment:
```suggestion
		// Unfold previous folded line on backspace if caret is at the first column.
```
Single line comments are preferred, even though block comments are used elsewhere in this file.
Same with below.